### PR TITLE
Use `add_common_op` for handling types and broadcast in Clip Onnx parsing

### LINKED ISSUE #1119: same_type: clip: Types do not match
I hit this error while running a test...

```
RuntimeError: /code/AMDMIGraphX/code/AMDMIGraphX/src/include/migraphx/check_shapes.hpp:115: same_type: clip: Types do not match
```

Paul mentioned that using `add_common_op` in parse_clip

### PR DESCRIPTION
**Pull request by umangyadav**
Should fix #1119

### CODE REVIEW

File: src/onnx/onnx_parser.cpp [hunk 1]
```diff
@@ -98,7 +98,13 @@ instruction_ref onnx_parser::node_info::add_broadcastable_binary_op(const std::s
                                                                     instruction_ref arg0,
                                                                     instruction_ref arg1) const
 {
-    return add_common_op(*mod, make_op(op_name), {arg0, arg1});
+    return migraphx::add_common_op(*mod, make_op(op_name), {arg0, arg1});
```
Comment (pfultz2) [inline src/onnx/onnx_parser.cpp:101]: This can be `this->add_common_op(make_op(op_name), {arg0, arg1})`

**Comment by pfultz2** [comment]
Tests need to be added for this.

**Comment by umangyadav** [comment]
> Tests need to be added for this.

@pfultz2  Added test.

File: src/onnx/onnx_parser.cpp (hunk 1 above)
Comment (umangyadav) [reply src/onnx/onnx_parser.cpp]: Done

**Review by pfultz2 — APPROVED** [review]

**Review by turneram — APPROVED** [review]

**Pull request merged**